spec/controllers/news_controller_spec.rb

```diff
@@ -67,7 +67,7 @@
       expect(response).to be_successful
       expect(response).to render_template "show"
 
-      expect(response.body).to have_css("h1", text: news.title)
+      expect(response.body).to have_css("h2", text: news.title)
```

Comment:
You could also use the `have_heading` matcher here:

```suggestion
      expect(response.body).to have_heading news.title
```

That said, controller specs shouldn't really concern themselves with what is rendered. I'd suggest removing `render_views` and replacing this with:

```suggestion
      expect(assigns(:news).title).to eq news.title
```

or better yet:

```suggestion
      expect(assigns(:news)).to eq news
```

- just a sanity check to ensure the ivar is defined correctly for the view.